Sources/protoc-gen-grpc-swift/Generator-Client+AsyncAwait.swift

```diff
@@ -95,49 +95,79 @@ extension Generator {
         self.method = method
 
         let rpcType = streamingType(self.method)
-        let callType = Types.call(for: rpcType)
-        let callTypeWithoutPrefix = Types.call(for: rpcType, withGRPCPrefix: false)
+        printRpcFunctionImplementation(rpcType: rpcType)
+        printRpcFunctionWrapper(rpcType: rpcType)
```

Comment:
nit: these should both be `printRPC...`

Reply:
Not sure I understand the intention of this comment.